docs/guidelines/components/tables.md

```diff
@@ -0,0 +1,202 @@
+    <tbody>
+        <tr>
+            <td class="ez-checkbox-cell">
+                <input type="checkbox">
+            </td>
+            <td>1</td>
+            <td>English (United Kingdom)</td>
+            <td>Administrator User</td>
+            <td>
+                <button type="button" class="btn btn-icon">
+                    <svg class="ez-icon ez-icon-edit">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="../../ez-icons.svg#archive-restore"></use>
+                    </svg>
+                </button>
+            </td>
+        </tr>
+    </tbody>
+</table>
+code_example]]
+</div>
+
+##<div class="mgt-3 header-line">Secondary table</div>
+<div class="ez-guidelines-tables__with-header mgb-5">
+[[code_example {html}
+<div class="ez-table-header ground-base">
+    <div class="ez-table-header__headline">Location Content Swap</div>
+</div>
+<table class="table ez-table--no-border">
+    <tbody>
+        <tr>
+            <td>Swap the content item at this location with another
+                <button type="button" class="btn btn-outline-secondary ml-5">Select content item</button>
+            </td>
+        </tr>
+    </tbody>
+</table>
+code_example]]
+</div>
+
+##<div class="mgt-3 header-line">Notifications table</div>
+<div class="ez-guidelines-tables__notifications mgb-5">
+[[code_example {html}
+<table class="table n-table--notifications">
+    <thead>
+        <tr>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Date</th>
+        </tr>
+    </thead>
+    <tbody class="n-table__body">
+        <tr class="n-notifications-modal__item fw-notification">
+            <td class="n-notifications-modal__type">
+                <span class="type__icon">
+                    <svg class="ez-icon ez-icon--review">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="../../ez-icons.svg#review"></use>
+                    </svg>
+                </span>
+                <span class="type__text">
+                    Content review request
+                </span>
+            </td>
+            <td class="n-notifications-modal__description">
+                <p class="description__title">
+                    From:
+                    <span class="description__title__item">Administrator user</span>
+                </p>
+                <p class="description__text">Can you check this? Thxs!</p>
+                <span class="description__read-more">Read more &raquo;</span>
+            </td>
+            <td class="n-notifications-modal__time">
+                September 13, 2018 14:58
+            </td>
+        </tr>
+    </tbody>
+</table>
+code_example]]
+</div>
+
+!!! Note
+    This table is displayed within `.n-notifications-modal` modal. Styling relies on this class.
+
+
+##<div class="mgt-3 header-line">List table</div>
+<div class="ez-guidelines-tables__list mgb-5">
+[[code_example {html}
+<table class="table ez-table--list">
```

Comment:
It should be `table ez-table ez-table--list`

Reply:
Good call @sunpietro. It's true (and I agree with you) that it should be that way, but it's what we currently have:

![screen shot 2018-09-14 at 3 39 07 pm](https://user-images.githubusercontent.com/9256718/45571556-99074200-b834-11e8-940e-9c5bdc3aa987.png)

This is also a question for you @DominikaK, Shouldn't we reflect here in the Guidelines the current status of each component? Or the standardized one? I don't have any problem updating the PR, but if this section of the documentation is going to be updated based on releases we should reflect that too. I'd go for this final option. But I need to hear you guys first.

Reply:
As a rule we do not name versions in the text, but instead merge changes to proper version branches. So if there is a change that will not apply to 2.2, please add it in a separate PR (to master now, but with a note which versions(s) is should go to).

Reply:
Updated to `table ez-table ez-table--list` as requested.